WindowsBuilder.ps1

```diff
@@ -0,0 +1,250 @@
+
+	Write-Yellow "Downloading MPG123 DLL..."
+	$f = "$([System.IO.Path]::GetFileName($url_mpg123))"
+	$wc.DownloadFile("$url_mpg123", "$downloads\$f")
+	7z e -o"$releasedir" "$f" "mpg123*/libmpg123*.dll"
+	7z e -o"$releasedir" "$f" "mpg123*/libmpg123*.def"
+	7z e -o"$includedir" "$f" "mpg123*/*.h"
+	7z e -o"$releasedir" "$f" "mpg123*/COPYING.txt"
+	cp "$releasedir\COPYING.txt" "$releasedir\LICENSE.MPG123"
+	rm -Force "$releasedir\COPYING.txt"
+
+	Write-Yellow "Creating MPG123 lib..."
+	cd "$releasedir"
+	mv "libmpg123-0.dll.def" "libmpg123-0.def"
+	lib /def:"libmpg123-0.def" /out:"$libdir\libmpg123-0.lib" /machine:"$arch"
+	rm "libmpg123-0.def"
+
+	Write-Yellow "Downloading sndfile..."
+	cd "$downloads"
+	$f = "$([System.IO.Path]::GetFileName($url_libsndfile))"
+	$wc.DownloadFile("$url_libsndfile", "$downloads\$f")
+	7z e -o"$libdir" "$f" "lib/*.lib" -r
+	7z e -o"$includedir" "$f" "include/*" -r
+	7z e -o"$releasedir" "$f" "bin/*.dll"
+
+	Write-Yellow "Downloading SDL2..."
+	$f = "$([System.IO.Path]::GetFileName($url_sdl2))"
+	$wc.DownloadFile("$url_sdl2", "$downloads\$f")
+	7z e -o"$libdir" "$f" "SDL2-*/lib/$arch/*.lib" -r
+	7z e -o"$includedir" "$f" "SDL2-*/include/*" -r
+	7z e -o"$releasedir" "$f" "SDL2-*/COPYING.txt" "SDL2-*/lib/$arch/*.dll"
+	cp "$releasedir\COPYING.txt" "$releasedir\LICENSE.SDL2"
+	rm -Force "$releasedir\COPYING.txt"
+
+	Write-Yellow "Downloading libuv..."
+	git clone "$url_libuv"
+	Write-Yellow "Compiling libuv..."
+	cd "libuv"
+	cmd /c "vcbuild.bat" "$arch" "release" "shared"
+	cp "Release/*.lib" "$libdir/"
+	cp "include/*" "$includedir/"
+	cp "Release/*.dll" "$releasedir/"
+	cp "LICENSE" "$releasedir/LICENSE.libuv"
+	cd "$oldpwd"
+}
+
+function BuildPlayd ($arch, $archdir, $build)
+{
+	Write-Yellow "Building ury-playd on $arch..."
+	$oldpwd = $pwd
+	cd "$build"
+
+	switch($arch) {
+		"x86" { $cmake_generator = "Visual Studio 14 2015"; $msbuild_platform = "Win32" }
+		"x64" { $cmake_generator = "Visual Studio 14 2015 Win64"; $msbuild_platform = "x64" }
+	}
+	Write-Yellow "Running cmake..."
+	cmake "$project" -G "$cmake_generator" -DCMAKE_PREFIX_PATH="$archdir"
+	Write-Yellow "Running msbuild..."
+	msbuild playd.sln /p:Configuration="Release" /toolsversion:14.0 /p:Platform="$msbuild_platform" /p:PlatformToolset=v140
+	cd "$oldpwd"
+}
+
+function Load-MSVC-Vars
+{
+	#Set environment variables for Visual Studio Command Prompt
+	pushd "$env:VS140COMNTOOLS"
+	cmd /c "vsvars32.bat&set" |
+	foreach {
+	  if ($_ -match "=") {
+		$v = $_.split("="); set-item -force -path "ENV:\$($v[0])"  -value "$($v[1])"
+	  }
+	}
+	popd
+	Write-Yellow "Visual Studio 2015 Command Prompt variables set."
+}
+
+# Check if an exe is Python 2.7.
+function Check-Python-Version ($pypath, $vstring)
+{
+	$python_version = & "$pypath" "--version" 2>&1
+	return $python_version -Match $vstring
+}
+
+# Check for Python 2.7, and throw an exception if not successful.
+function Check-Python ($vstring)
+{
+	if (Test-Path Env:\PYTHON)
+	{

```

Comment:
Is this style of if {} else {} required in powershell? I'd prefer it if the opening/closing braces were on the same line, like the cpp code